legend-pure-dsl/legend-pure-dsl-tds/legend-pure-m2-dsl-tds-grammar/src/main/java/org/finos/legend/pure/m2/inlinedsl/tds/TDSExtension.java

```diff
@@ -236,17 +237,22 @@ private static String convertType(DataType dataType)
         }
     }
 
-    private static SinkFactory makeMySinkFactory()
+    public static CsvSpecs makePureCsvSpecs()
+    {
+        return CsvSpecs.builder().nullValueLiterals(Arrays.asList("", "null")).build();
```

Comment:
When parsing, either empty or the literal null will be treated as null values.  This is mimicking what we already do on Engine.